src/test/java/bio/terra/service/filedata/azure/tables/LoadHistoryStorageTableConnectedTest.java

```diff
@@ -0,0 +1,158 @@
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
+@Category(Connected.class)
+public class LoadHistoryStorageTableConnectedTest {
+  private static final String MANAGED_RESOURCE_GROUP_NAME = "mrg-tdr-dev-preview-20210802154510";
+  private static final String STORAGE_ACCOUNT_NAME = "tdrshiqauwlpzxavohmxxhfv";
+
+  private UUID applicationId;
+  private UUID storageAccountId;
+  private UUID datasetId;
+
+  private AzureApplicationDeploymentResource applicationResource;
+  private AzureStorageAccountResource storageAccountResource;
+  private BillingProfileModel billingProfile;
+
+  @Autowired AzureSynapsePdao azureSynapsePdao;
+  @Autowired ConnectedOperations connectedOperations;
+  @Autowired private ConnectedTestConfiguration testConfig;
+  @Autowired DatasetService datasetService;
+  @MockBean private IamProviderInterface samService;
+  @Autowired SynapseUtils synapseUtils;
+  @Autowired AzureAuthService azureAuthService;
+  @Autowired TableDirectoryDao tableDirectoryDao;
+  @Autowired FileMetadataUtils fileMetadataUtils;
+  @Autowired TableDao tableDao;
+  @Autowired AzureBlobStorePdao azureBlobStorePdao;
+  @Autowired FileService fileService;
+  @Autowired AzureStorageTablePdao storageTableDao;
+
+  @Before
+  public void setup() throws Exception {
+    connectedOperations.stubOutSamCalls(samService);
+    applicationId = UUID.randomUUID();
+    storageAccountId = UUID.randomUUID();
+    datasetId = UUID.randomUUID();
+
+    billingProfile =
+        new BillingProfileModel()
+            .id(UUID.randomUUID())
+            .profileName(Names.randomizeName("somename"))
+            .biller("direct")
+            .billingAccountId(testConfig.getGoogleBillingAccountId())
+            .description("random description")
+            .cloudPlatform(CloudPlatform.AZURE)
+            .tenantId(testConfig.getTargetTenantId())
+            .subscriptionId(testConfig.getTargetSubscriptionId())
+            .resourceGroupName(testConfig.getTargetResourceGroupName())
+            .applicationDeploymentName(testConfig.getTargetApplicationName());
+
+    applicationResource =
+        new AzureApplicationDeploymentResource()
+            .id(applicationId)
+            .azureApplicationDeploymentName(testConfig.getTargetApplicationName())
+            .azureResourceGroupName(MANAGED_RESOURCE_GROUP_NAME)
+            .profileId(billingProfile.getId());
+    storageAccountResource =
+        new AzureStorageAccountResource()
+            .resourceId(storageAccountId)
+            .name(STORAGE_ACCOUNT_NAME)
+            .applicationResource(applicationResource)
+            .metadataContainer("metadata")
+            .dataContainer("data");
+  }
+
+  @After
+  public void cleanup() throws Exception {
+    // TODO - clean up the load history table
```

Comment:
I can just create a follow-on ticket to clean this and the other connected tests up. Apologies these weren't polished before I left 😞